File: src/main/java/br/com/jiratorio/repository/IssueCustomRepository.java
package br.com.jiratorio.repository;

import br.com.jiratorio.domain.dynamicfield.DynamicFieldsValues;
import br.com.jiratorio.domain.entity.Issue;
import br.com.jiratorio.domain.form.IssueForm;
import java.util.List;

public interface IssueCustomRepository {

    List<Issue> findByExample(Long boardId, IssueForm issueForm);

    List<DynamicFieldsValues> findAllDynamicFieldValues(Long boardId);

}

File: src/main/java/br/com/jiratorio/repository/IssueCustomRepository.kt
package br.com.jiratorio.repository

import br.com.jiratorio.domain.dynamicfield.DynamicFieldsValues
import br.com.jiratorio.domain.entity.Issue
import br.com.jiratorio.domain.request.SearchIssueRequest

interface IssueCustomRepository {

    fun findByExample(boardId: Long, searchIssueRequest: SearchIssueRequest): List<Issue>

    fun findAllDynamicFieldValues(boardId: Long): List<DynamicFieldsValues>

}
